### DISCUSSION TITLE
Is Behaviour Metrics Repo still taking contributions?

### DISCUSSION THREAD

**Discussion by AAK121** [JdeRobot/jderobot.github.io#285] [General]
My name is Alok, and I am a second-year graduate student at IIT Bombay with a deep interest in Deep Learning. I am eager to contribute to the field and am particularly interested in contributing to your Behaviour Metrics repository. However, I noticed that the last commits were made around 3-5 years ago, which has left me uncertain about whether the repository is still actively maintained and accepting contributions.

I also introduced myself in the discussions section of the repository but have not received any response yet. I would appreciate any guidance or clarification regarding the repository’s current status and whether it is still open to contributions.

Additionally, if there are any other repositories, preferably related to ML/DL, that you would recommend for contribution, I would be more than happy to explore those as well.

**Comment by PIKABOLTCODER** [comment]
Hi Alok,

I’m Madhav, a first-year undergraduate student at Visvesvaraya National Institute of Technology (VNIT), Nagpur. I’ve also recently started exploring open-source contributions, and I’m currently working with the JdeRobot repository.

I noticed your interest in contributing to Behaviour Metrics, and while I’m not directly working on that, I believe JdeRobot could be a great repository to explore together. It’s related to robotics and artificial intelligence, and it provides interesting challenges that align well with ML/DL concepts.

If you’re interested, maybe we could collaborate on contributing to the JdeRobot repository. It could be a great way to learn, share ideas, and make an impact in the open-source community. Let me know what you think!

Looking forward to hearing from you.
